src/vos/vos_pool.c

```diff
@@ -679,6 +690,22 @@ pool_open(void *ph, struct vos_pool_df *pool_df, unsigned int flags, void *metri
 		D_GOTO(failed, rc);
 	}
 
+	xs_ctxt = vos_xsctxt_get();
+	D_DEBUG(DB_MGMT, "Opening VOS I/O context for xs:%p pool:"DF_UUID"\n",
+		xs_ctxt, DP_UUID(pool_df->pd_id));
+	if (pool_df->pd_nvme_sz == 0)
+		bm_flags |= BIO_MC_FL_NO_DATABLOB;
+
+	rc = bio_mc_open(xs_ctxt, pool_df->pd_id, &pool->vp_umm,
```

Comment:
bio_mc_open() needs be done before we get "pool_df".

Reply:
yes, my mind was broken..